### DISCUSSION TITLE
live (or close to live) sync for shopping list

### DISCUSSION THREAD

**Discussion by g-vdn** [mealie-recipes/mealie#1715] [Feature Request]
Would be great to have a live update or a sync each 5 secs or something on the shopping list. Like OurGrocery app.
You go to the market with somebody else - both of you can "work" on same shopping list, checking items and stuff. Check item on phone A (account A), after x amount of seconds, that same item is checked also on phone B (account B).

Thanks!

P.S. Amazing looking app so far, now I'm checking how to switch from Cookbook/Nextcloud to this one.

**Comment by michael-genson** [comment]
Interesting. So, for example, two people have the same shopping list open on two different devices, and if one person checks off an item, the other person sees that change?

**Reply by g-vdn** [reply]
Yes, exactly. You can check out for reference the "OurGrocery" app on PlayStore. The one thing with OurGrocery app is you need to log-in with same username on all devices. I tried this on Mealie, 2 different users, 1 on desktop one on phone, both users being part of same group and having same list opened. One user would have to refresh the page before checking some item, so it will have the list updated. Not experienced in programming, but couldn't be selected an event trigger? Like when the state of the list is changed, all clients to refresh page automatically.

Thanks

**Reply by michael-genson** [reply]
Thanks! I just submitted a PR for this: #1730
